src/compiler/python_generator.cc

```diff
@@ -397,12 +439,13 @@ bool PrintBetaStubFactory(const grpc::string& package_qualified_service_name,
                                    &output_message_module_and_class)) {
         return false;
       }
+      grpc::string method_name = ToSnakeCase(method->name());
       method_cardinalities.insert(
-          make_pair(method->name(), method_cardinality));
+          make_pair(method_name, method_cardinality));
       input_message_modules_and_classes.insert(
-          make_pair(method->name(), input_message_module_and_class));
+          make_pair(method_name, input_message_module_and_class));
```

Comment:
Same as above.  I *think* mangling the names for cardinalities is correct though.